components/02-elements/table/_table.scss

```diff
@@ -27,89 +27,109 @@ $table-radius: $border-radius;
             }
         }
     }
-        
+
+    thead {
+        tr {
+            padding: 0;
+            border-bottom: 0;
+
+            @include mq($screen-m) {
+                border-top-left-radius: $table-radius;
+                border-top-right-radius: $table-radius;
```

Comment:
is it needed? You've added border-radius to table there is no need to add to every elements, b/c is nested :) Or maybe you've notice some issue concerning other browsers? I've checked on Chrome, FF, safari and it works with border-radius only on `<table>` element. Please correct and check.

Reply:
Without this border-radius
![This](https://user-images.githubusercontent.com/11212541/30207899-77011022-9491-11e7-97dd-8cf6482f33fc.png)

With border-radius
![And this](https://user-images.githubusercontent.com/11212541/30207915-8a6084d6-9491-11e7-9f4d-787bbf865d39.png)

Reply:
that's why they are here and I spent some time on this to property control this. The simplest solution is to delete this radius than I can delete this all code

Reply:
Yup. You're right. I haven't noticed it earlier. Now I see it for bottom corners :/

Reply:
fixed in tbody too